src/controllers/EventsController.ts

```diff
@@ -1,85 +1,91 @@
-import { NextFunction, Request, Response } from 'express'
+import { Request, Response } from 'express'
 import querySchema from '../validation/addEventValidate'
 import { Message } from '../config/messages'
 import { Event, User } from '../db'
 import { Op } from 'sequelize'
+import CustomError from '../helpers/CustomError'
 
 export default class EventsController {
   // for getting all data
-  public static async index (req: Request, res: Response, next: NextFunction) {
+  public static async index (req: Request, res: Response) {
```

Comment:
You may need to specify the return type of the function
```suggestion
  public static async index (req: Request, res: Response):Promise<void> {
```